src/SFA.DAS.EmploymentCheck.Functions/Application/Services/Hmrc/HmrcService.cs

```diff
@@ -71,7 +71,7 @@ AzureServiceTokenProvider azureServiceTokenProvider
                 if (result != null)
                 {
                     request.Employed = result.Employed;
-                    request.RequestCompletionStatus = 200;
+                    //request.RequestCompletionStatus = 200;
```

Comment:
Presumably this is commented out because the error code is hard set in the object creation below. If that's the case, can this be removed please?